x/oracle/abci_test.go

```diff
@@ -240,4 +250,14 @@ func (suite abciTestSuite) TestEndBlockerVoteReject() {
 	oracleVotes, err = suite.OracleKeeper.GetAllOracleRegistrationVoteList(ctx)
 	suite.Require().NoError(err)
 	suite.Require().Equal(0, len(oracleVotes))
+
+	events := ctx.EventManager().Events()
```

Comment:
It's not a big issue. But you can change it to `suite.Ctx.EventManager().Events()`.

Reply:
Yes, I used the `ctx` variable from here to `suite.Ctx`

https://github.com/medibloc/panacea-core/pull/391/files#diff-5ab93a9d86b554f353a4615e22adbfef295849e090c0b139aae4fb37bf04f499R159